data/card_catalogue.json

```diff
@@ -355,25 +417,25 @@
           "upgrade": "",
           "tier": "blue",
           "details": {
-            "rules": "Deals 10 damage, breaking on use",
+            "rules": "Deals 10 damage. Return this card to the deck after using it.",
```

Comment:
This is the one time that I'll allow the `return to the deck` mechanic

Reply:
Any thoughts on switching to "cast" wording to avoid it?

Reply:
How does that avoid it? Agreed that using cast instead of use is an improvement though, I'll re-write this

Reply:
Just as cast carries the implication of expending the card - I suppose I should have first asked if you think it's important that we return it to the deck in lieu of having it behave like a normal cast?